### ISSUE TITLE
Global Search Not Working

### ISSUE THREAD

**Issue by mdavis1982** [optimistdigital/nova-sortable#21]
Hi @Tarpsvo 

I've just noticed that with the latest release, my global search isn't working, and I seem to be getting errors from the `HasSortableRows` trait.

My setup is detailed in #20 that we worked through over the last couple of days.

I'm getting "No Results" in the global search.

This doesn't actually seem to be related to using a `belongsToMany` ordering as this happens even when I only have a standard sorting happening on a single table.

These are the errors I'm seeing in my stack trace:

```json
{
    "file": "/Users/matt/Code/Sites/redacted/vendor/optimistdigital/nova-sortable/src/Traits/HasSortableRows.php",
    "line": 13,
    "function": "newResource",
    "class": "Laravel\\Nova\\Http\\Requests\\NovaRequest",
    "type": "->"
},
{
    "file": "/Users/matt/Code/Sites/redacted/vendor/optimistdigital/nova-sortable/src/Traits/HasSortableRows.php",
    "line": 59,
    "function": "getSortability",
    "class": "App\\Nova\\Resources\\Tag",
    "type": "::"
},
```

Any help would be greatly appreciated! Thanks 😄

**Comment by Tarpsvo** [comment]
Hey! Fixed the issue and released in version 1.4.5. Good luck!

**Comment by mdavis1982** [comment]
@Tarpsvo Works a treat! Thanks you so much for your support on this. You really should set up GitHub Sponsors 😄
